low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -410,77 +440,94 @@ def _generate_host_update(self):
-                                    self._node_sensor.up_time,
-                                    self._node_sensor.uname, self._units,
-                                    self._node_sensor.total_memory,
-                                    self._node_sensor.logged_in_users,
-                                    self._node_sensor.process_count,
-                                    self._node_sensor.running_process_count,
-                                    self.FAULT,
-                                    fault_event
-                                    )
-            # Add in uuid if it was present in the json request
-            if self._uuid is not None:
-                hostUpdateMsg.set_uuid(self._uuid)
-            jsonMsg = hostUpdateMsg.getJson()
-            # Transmit it to message processor
-            self.host_sensor_data = jsonMsg
-            self.os_sensor_type["memory_usage"] = self.host_sensor_data
-            self._write_internal_msgQ(EgressProcessor.name(), jsonMsg)
-            self.persist_state_data('memory', 'MEMORY_USAGE_DATA')
-
-        if self._node_sensor.total_memory["percent"] < self._host_memory_usage_threshold \
-           and self.high_usage['memory']:
-
-            fault_resolved_event = "Host memory usage decreased to %s, lesser than configured threshold of %s" \
-                                    %(self._node_sensor.total_memory["percent"],
-                                      self._host_memory_usage_threshold)
-            logger.warning(fault_resolved_event)
-            logged_in_users = []
-            # Create the host update message and hand it over to the egress processor to transmit
-            hostUpdateMsg = HostUpdateMsg(self._node_sensor.host_id,
-                                    self._epoch_time,
-                                    self._node_sensor.boot_time,
-                                    self._node_sensor.up_time,
-                                    self._node_sensor.uname, self._units,
-                                    self._node_sensor.total_memory,
-                                    self._node_sensor.logged_in_users,
-                                    self._node_sensor.process_count,
-                                    self._node_sensor.running_process_count,
-                                    self.FAULT_RESOLVED,
-                                    fault_resolved_event
-                                    )
-
-            # Add in uuid if it was present in the json request
-            if self._uuid is not None:
-                hostUpdateMsg.set_uuid(self._uuid)
-            jsonMsg = hostUpdateMsg.getJson()
-            # Transmit it to message processor
-            self.host_sensor_data = jsonMsg
-            self.os_sensor_type["memory_usage"] = self.host_sensor_data
-            self._write_internal_msgQ(EgressProcessor.name(), jsonMsg)
-            self.high_usage['memory'] = False
-            self.persist_state_data('memory', 'MEMORY_USAGE_DATA')

```

Comment:
why it is showing all of the lines are removed? may be because of code alignment?

Reply:
Yes, I have added an outer if condition, so inner code has to be aligned. That's why it is showing all these lines as changes.